case-studies/aepsi.html

```diff
@@ -0,0 +1,123 @@
+---
+title: AEPSi Case Study
+description: SmartLogic build of web application for AEPSi.
+body-class: case-study
+breadcrumb: AEPSi
+---
+<main class="main" role="main">
+    <section class="section hero hero-no-img">
+      <div class="container">
+        <div class="hero__left">
+          <div class="hero__left-text">
+            <div class="head-text">
+              <h1>AEPSinteractive</h1>
```

Comment:
It would be good to define AEPS either in the header or in the first paragraph:

[Assessment, Evaluation, and Programming System for Infants and Children](https://brookespublishing.com/product/aeps/)

Reply:
@type1fool That threw me too, but that is how it's branded on the brookes website